pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -701,7 +701,11 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 		}
 
 		// Execute mount
-		mountErr := volumeMounter.SetUp(fsGroup)
+		mountErr := volumeMounter.SetUp(volume.MounterArgs{
+			FsGroup:     fsGroup,
+			DesiredSize: volumeToMount.DesiredSizeLimit,
+			PodUID:      string(volumeToMount.Pod.UID),
```

Comment:
You don't have to add this argument here. It should be available via the `emptyDir` data structure in the `SetUp` method

Reply:
If you want, I can remove it; I agree it's not really needed here.

Reply:
As for accumulating quotas, it's not impossible that something could go wrong with the teardown and leave the quota in place applying to nothing.  It is possible to remove quotas later if need be.  We leave records in /etc/projects and /etc/projid with clearly defined names to make it possible to trace back to Kubernetes.

Reply:
> If you want, I can remove it; I agree it's not really needed here.

If you have time go for it, happy to reapply lgtm/approval. If not, no big deal.